tests/client/doc.scan.client.spec.js

```diff
@@ -0,0 +1,120 @@
+const fs = require('fs');
+const nock = require('nock');
+const uuid = require('uuid');
+
+const config = require('../../config');
+
+const {
+  DocScanClient,
+  SessionSpecificationBuilder,
+} = require('../../');
+
+const SessionResult = require('../../src/doc_scan_service/session/create/create.session.result');
```

Comment:
Should we change this to `CreateSessionResult` so it's named the same as the actual class you're requiring?

Reply:
Fixed in 46e3c65